src/main/xmainWindow.js

```diff
@@ -0,0 +1,79 @@
+const path = require('path');
+
+const express = require('express');
+const {router} = require('./xserver/xroutes/xroutes.js');
+const exApp = express();
+const bodyParser = require('body-parser');
+
+const { app, BrowserWindow } = require('electron');
+const isDev = require('electron-is-dev');
+const os = require('os');
+require('electron-reload')(__dirname);
+
+
+let PORT = 3002;
+let serverIsOn = true;
+let itemsToSend; //(FOR DESTINATION PANEL)
+
+
+
+
+let mainWindow;
+function window() {
+
+  exApp.use(bodyParser.urlencoded({ encoded: true }))
+  exApp.use(bodyParser.json());
+
+
+  exApp.use('/', router);
+
+  mainWindow = new BrowserWindow({
+    width: 1200,
+    height: 900,
+    webPreferences: {
+      nodeIntegration: true,
+      webSecurity: false,
+    },
+    icon: '../../../index_png.png',
+  });
+
+  mainWindow.loadURL(isDev ? 'http://localhost:8080' : `file://${__dirname}/../../dist/index.html`);
+
+  if (isDev) {
+    BrowserWindow.addDevToolsExtension(
+      path.join(os.homedir(), './Library/Application Support/Google/Chrome/Default/Extensions/fmkadmapgofadopljbjfkapdkoienihi/4.0.6_0'),
```

Comment:
4.1?

Reply:
I believe that whole extensions folder name needs to change because I didn't have that folder originally. That folder was slacked to me by Derek awhile back, but then the folder disappeared from my files for some reason. I'll change it once I know what the new folder is called.